### DISCUSSION TITLE
SkBee does not load on 1.16.5

### DISCUSSION THREAD

**Discussion by karamzyatina** [ShaneBeee/SkBee#321] [Q&A]
Hello everyone I have a problem. I need to add a couple of lines in the Script 2.6.4 plugin that will remove the commands from the Tab Compliter list. I downloaded your extension and uploaded it to the server. For some reason, it doesn't work. Maybe this is due to 1.16.5? If so, what version of your extension do I need to download?
I may also need your help in writing these few lines, but it's not important yet.

**Comment by karamzyatina** [comment]
![Tabby_Yv0wsI5chp](https://user-images.githubusercontent.com/67735224/212535141-9d9994ef-a6b4-4e1b-842b-5f0e7e03be8d.png)

**Comment by ShaneBeee** [comment]
If you require help using SkBee, I recommend joining the [**skUnity discord**](https://discord.com/invite/0l3WlzBPKX7WNjkf)
This is not the place to get help.
This is an issue tracker for the current version of SkBee.
